### PR TITLE
Map Warp

### PR DESCRIPTION
**Pull request by SpockBauru**
map warp map a warp to map thingy

### CODE REVIEW

File: Translation/en/Text/Main/Generic.txt [hunk 1]
```diff
@@ -101,10 +108,19 @@
 水着=Swimsuit
 寝間着=Pajamas
 風呂場=Bathing
+お風呂=Bathing
```
Comment (ManlyMarco) [inline Translation/en/Text/Main/Generic.txt:111]: This should be `Bath`

File: Translation/en/Text/Main/Menu-FreeH.txt [hunk 2]
```diff
@@ -83,9 +83,9 @@ H段階=H-Level
 
 //Time of Day
 時間帯=Time Of Day
-朝=Morning
+朝=AM
```
Comment (ManlyMarco) [inline Translation/en/Text/Main/Menu-FreeH.txt:86]: Dawn

File: Translation/en/Text/Main/Menu-FreeH.txt [hunk 3]
```diff
@@ -83,9 +83,9 @@ H段階=H-Level
...
 昼=Noon
-夕方=Evening
+夕方=PM
```
Comment (ManlyMarco) [inline Translation/en/Text/Main/Menu-FreeH.txt:88]: Dusk

File: Translation/en/Text/Main/Menu-GirlsRoom.txt [hunk 4]
```diff
@@ -60,9 +60,17 @@
...
 
+//Change Girl's Location
+移動中=Moving
+ここに移動=Go Here
+入れ替える=Replace
+
 プレイヤーの愛称=Player's Nickname
 変更=Change
 
+全解除=All
```
Comment (ManlyMarco) [inline Translation/en/Text/Main/Menu-GirlsRoom.txt:71]: Cancel All

File: Translation/en/Text/Main/Menu-GirlsRoom.txt [hunk 5]
```diff
@@ -102,6 +110,16 @@
 // Player Menu
 //---------------------------------------------------------
 プレイヤー情報=Player Info
+呼ばれる愛称=Nickname Called
```
Comment (ManlyMarco) [inline Translation/en/Text/Main/Menu-GirlsRoom.txt:113]: Called-by nickname? Literally it's `A nickname to be called`

File: Translation/en/RedirectedResources/assets/abdata/map/list/mapinfo/00/00/translation.txt [hunk 6]
```diff
@@ -1,39 +1,39 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-ホテル（外観）=Hotel (Exterior)
-民宿（外観）=Guest House (Exterior)
-合宿所（外観）=Boarding House (Exterior)
+ホテル（外観）=Hotel Exterior
+民宿（外観）=Guest House Exterior
+合宿所（外観）=Training Camp Exterior
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/map/list/mapinfo/00/00/translation.txt:6]: Still not decided on how to call this area

File: Translation/en/Text/Localizations/Instance/TimeUIControl.txt [hunk 7]
```diff
@@ -1,7 +1,7 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-朝=Morning
-昼=Midday
-夕=Evening
+朝=AM
+昼=Noon
+夕=PM
```
Comment (ManlyMarco) [inline Translation/en/Text/Localizations/Instance/TimeUIControl.txt:6]: Dawn
Noon
Dusk

File: Translation/en/Text/Main/ActionScene.txt [hunk 8]
```diff
@@ -115,7 +115,7 @@ r:^※コイカツポイントを(\d+)pt消費します。$=※ Consumes $1 Koik
 祈り結果=Prayer
 個数=Count
 入手=Found
-入手話題=Obtained Topics
+入手話題=Topics Inventory
```
Comment (ManlyMarco) [inline Translation/en/Text/Main/ActionScene.txt:118]: Obtained Topics / Available Topics is better

**Review by ManlyMarco — CHANGES_REQUESTED** [review]
We need to figure out how to call the training camp

File: Translation/en/Text/Main/Menu-GirlsRoom.txt (hunk 4 above)
Comment (SpockBauru) [reply Translation/en/Text/Main/Menu-GirlsRoom.txt:71]: Hmm... This one clear all traits when you select a girl on the room, so I think that "Clear All" is better

File: Translation/en/Text/Main/Menu-FreeH.txt (hunk 3 above)
Comment (SpockBauru) [reply Translation/en/Text/Main/Menu-FreeH.txt:88]: Ok, will change to Dusk. If I have to change this one more time I'll let as twilight!

**Review by ManlyMarco — APPROVED** [review]

**Pull request merged**